cookbooks/aws-parallelcluster-slurm/recipes/config/login_nodes_daemon_service.rb

```diff
@@ -0,0 +1,54 @@
+require 'yaml'
+config = YAML.safe_load(File.read(node['cluster']['cluster_config_path']))
+GRACETIME_PERIOD = config["LoginNodes"]["Pools"][0]["GRACETIME_PERIOD"]
+
+# Create configuration file
+template "#{node['cluster']['shared_dir']}/login_nodes_daemon_config.json" do
+  source 'login_nodes_daemon_service/login_nodes_daemon_config.json.erb'
+  owner 'root'
+  group 'root'
+  mode '0644'
+  variables(
+    {
+      GRACETIME_PERIOD: GRACETIME_PERIOD,
+    }
+  )
+end
+
+# Create on_termination_script.sh
+template "#{node['cluster']['shared_dir']}/login_nodes_on_termination_script.sh" do
+  source 'login_nodes_daemon_service/login_nodes_on_termination_script.sh.erb'
+  owner 'root'
+  group 'root'
+  mode '0755'
+end
+
+# Create daemon_script.sh
+template "#{node['cluster']['shared_dir']}/login_nodes_daemon_script.sh" do
+  source 'login_nodes_daemon_service/login_nodes_daemon_script.sh.erb'
+  owner 'root'
+  group 'root'
+  mode '0755'
+end
+
+# Create supervisord configuration for the daemon
+file "/etc/supervisor/conf.d/login_nodes_daemon_service.conf" do
```

Comment:
We have a central supervisord configuration in `cookbooks/aws-parallelcluster-platform/templates/supervisord/parallelcluster_supervisord.conf.erb`.

I like your approach, but:
1. That way of configuration seems not supported, see http://supervisord.org/configuration.html
2. Even if it was supported, it would be better to be consistent with the approach we adopted so far; in case we disagree with the current approach, better to be consistent and open a follow up activity to revise it given we are close to the deadline.

Reply:
Have modified it, thank you for reminding!